WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -109,10 +107,7 @@ class NoticePresenter: NSObject {
                     return
             }
 
-            self.currentKeyboardPresentation = KeyboardPresentation(
-                isPresent: true,
-                height: keyboardFrameValue.cgRectValue.size.height
-            )
+            self.currentKeyboardPresentation = .present(height: keyboardFrameValue.cgRectValue.size.height)
```

Comment:
Do we need `self.` here?

Reply:
It's in the closure, so we need the self.